demos/crm-sharing/resources/jv1-add-contacts.gql

```diff
@@ -0,0 +1,76 @@
+mutation createAccountContacts {
+    contact1: add_AccountContact(
+        input: {
+            partner: "JV1",
+            accountName: "Vendia",
+            duns: "111111111",
+            primaryContactName: "Chris Smith",
+            primaryContactEmail: "not-real-1@vendia.net",
+            primaryContactPhone: "9085551212",
+            users: [
+                {
+                    name: "Robin Jones",
+                    role: "Account Executive",
+                    emailAddress: "not-real-2@vendia.net",
+                    phoneNumber: "9085551213"
+                },
+                {
+                    name: "Pat Singh",
+                    role: "Account Analyst",
+                    emailAddress: "not-real-3@vendia.net",
+                    phoneNumber: "9085551214"
+                }
+            ]
+        },
+        syncMode: ASYNC,
+        aclInput: {
+            acl: [
+                {principal: { nodes: [ "*" ] }, operations: [ READ ] }
+            ]
+        }
+    ) {
+        transaction {
+            _id
+        }
+    },
+
+    contact2: add_AccountContact(
+        input: {
+            partner: "JV1",
+            accountName: "Amazon",
+            duns: "222222222",
+            primaryContactName: "Terry Patel",
+            primaryContactEmail: "not-real-1@amazon.com",
+            primaryContactPhone: "2125551212",
+            users: [
+                {
+                    name: "Jo Anderson",
+                    role: "Account Executive",
+                    emailAddress: "not-real-2@amazon.com",
+                    phoneNumber: "2125551213"
+                },
+                {
+                    name: "Frank Palumbo",
+                    role: "Account Analyst",
+                    emailAddress: "not-real-3@amazon.com",
+                    phoneNumber: "2125551214"
+                }
+            ]
+        },
+        syncMode: ASYNC
+        aclInput: {
+            acl: [
+                {principal: { nodes: [ "JV2" ] }, path: "partner", operations: [ READ ] },
```

Comment:
Personal preference - I think it's helpful to include the (implicit) 1-liner ACL for the owner (in this case JV1).  Without that, the ACLs aren't comprehensive and may confuse the reader.  Thoughts on standardizing to show/to not show so our respective examples look the same?

Reply:
Good question - I think we can compromise and display the implicit ACL as a comment in GraphQL. Stay tuned for an update...

```
# Implicit ACL for the data writer - not explicitly needed
# but provided as a reference.
# {principal: { nodes: [ "JV1" ] }, operations: [ ALL, UPDATE_ACL ] }
```